roxctl/scanner/scanner.go

```diff
@@ -4,20 +4,21 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/roxctl/common/environment"
 	"github.com/stackrox/rox/roxctl/common/flags"
 	"github.com/stackrox/rox/roxctl/scanner/generate"
 	"github.com/stackrox/rox/roxctl/scanner/uploaddb"
 )
 
 // Command controls all of the functions being applied to a sensor
-func Command() *cobra.Command {
+func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c := &cobra.Command{
 		Use: "scanner",
 	}
 	flags.AddTimeoutWithDefault(c, time.Minute)
 	c.AddCommand(
 		generate.Command(),
```

Comment:
For the sake of consistency, it might be good to pass cliEnvironment to `generate.Command()` as well.
The parameter may not be used at the moment, but would be available later on when the need arises.

Reply:
✅